eng/scripts/verify-npm-tags.ps1

```diff
@@ -0,0 +1,62 @@
+param (
+  [Parameter(mandatory = $true)]
+  $artifactName,
+  [Parameter(mandatory = $true)]
+  $packageName
```

Comment:
I asse your one call is passing parameters like: `-packageName ${{parameters.Artifact.name}} -workingDirectory $(System.DefaultWorkingDirectory)` which doesn't match the parameters in this script currently.